src/main/java/uk/gov/ons/census/fwmt/jobservice/service/routing/nc/NcCeCreateEnglandAndWales.java

```diff
@@ -0,0 +1,104 @@
+package uk.gov.ons.census.fwmt.jobservice.service.routing.nc;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import uk.gov.ons.census.fwmt.common.data.tm.CaseRequest;
+import uk.gov.ons.census.fwmt.common.error.GatewayException;
+import uk.gov.ons.census.fwmt.common.rm.dto.ActionInstructionType;
+import uk.gov.ons.census.fwmt.common.rm.dto.FwmtActionInstruction;
+import uk.gov.ons.census.fwmt.events.component.GatewayEventManager;
+import uk.gov.ons.census.fwmt.jobservice.data.GatewayCache;
+import uk.gov.ons.census.fwmt.jobservice.http.comet.CometRestClient;
+import uk.gov.ons.census.fwmt.jobservice.service.GatewayCacheService;
+import uk.gov.ons.census.fwmt.jobservice.service.converter.nc.NcCreateConverter;
+import uk.gov.ons.census.fwmt.jobservice.service.processor.InboundProcessor;
+import uk.gov.ons.census.fwmt.jobservice.service.processor.ProcessorKey;
+import uk.gov.ons.census.fwmt.jobservice.service.routing.RoutingValidator;
+
+import java.time.Instant;
+import java.util.UUID;
+
+import static uk.gov.ons.census.fwmt.jobservice.config.GatewayEventsConfig.COMET_CREATE_ACK;
+import static uk.gov.ons.census.fwmt.jobservice.config.GatewayEventsConfig.COMET_CREATE_PRE_SENDING;
+import static uk.gov.ons.census.fwmt.jobservice.config.GatewayEventsConfig.FAILED_TO_CREATE_TM_JOB;
+
+@Qualifier("Create")
+@Service
+public class NcCeCreateEnglandAndWales implements InboundProcessor<FwmtActionInstruction> {
+
+  private static final ProcessorKey key = ProcessorKey.builder()
+      .actionInstruction(ActionInstructionType.CREATE.toString())
+      .surveyName("CENSUS")
+      .addressType("CE")
+      .addressLevel("E")
+      .build();
+
+  @Autowired
+  private CometRestClient cometRestClient;
+
+  @Autowired
+  private GatewayEventManager eventManager;
+
+  @Autowired
+  private RoutingValidator routingValidator;
+
+  @Autowired
+  private GatewayCacheService cacheService;
+
+  @Override
+  public ProcessorKey getKey() {
+    return key;
+  }
+
+  @Override
+  public boolean isValid(FwmtActionInstruction rmRequest, GatewayCache cache) {
+    try {
+      return rmRequest.getActionInstruction() == ActionInstructionType.CREATE
+          && rmRequest.getSurveyName().equals("CENSUS")
+          && !rmRequest.getOa().startsWith("N");
```

Comment:
Should there not be a check for an NC? Otherwise there are going to be more than one processor returned.
"When actionInstruction = "CREATE"
and surveyName = "CENSUS"
and not oa.startsWith('N')
and nc = true"